frame/contracts/src/lib.rs

```diff
@@ -177,8 +177,12 @@ pub mod pallet {
 	use frame_system::pallet_prelude::*;
 
 	/// The current storage version.
+	#[cfg(not(test))]
 	const STORAGE_VERSION: StorageVersion = StorageVersion::new(9);
```

Comment:
Thought maybe we can actually use  migration::VERSION_RANGE.1 here 🤔  to ensure the storage version is always in sync with the migration